src/components/Property/Types/VectorProperty/VectorProperty.tsx

```diff
@@ -20,11 +23,24 @@ export interface VectorPropertyProps {
 }
 
 export function VectorProperty(props: VectorPropertyProps) {
-  const { value, viewOptions } = props;
+  const { value, viewOptions, name, description } = props;
+
+  const isColor = (value.length === 3 || value.length === 4) && viewOptions.Color;
+  const isMinMaxRange = value.length === 2 && viewOptions.MinMaxRange;
 
-  if (value.length === 2 && viewOptions.MinMaxRange) {
-    return <MinMaxRange {...props} />;
+  let view;
+  if (isColor) {
+    view = <ColorVector {...props} />;
+  } else if (isMinMaxRange) {
+    view = <MinMaxRange {...props} />;
+  } else {
+    view = <ValueList {...props} />;
   }
 
-  return <ValueList {...props} />;
+  return (
+    <>
+      <PropertyLabel label={name} tip={description} />
+      {view}
+    </>
+  );
```

Comment:
Make this a switch that uses viewOptions and make each case return?

Reply:
Can't make a switch, but I'll make if statements for each view option, that directly returns